js/admin/image.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+$(function(){
+  const checkboxName = 'PS_IMAGE_FORMAT[]';
+  const formId = 'image_type_form';
+  const checkedCheckboxes = `input[name="${checkboxName}"]:checked`
+
+  // on page load, disable checkbox if there is only one checked
+  if (document.querySelectorAll(checkedCheckboxes).length === 1) {
+    document.querySelector(checkedCheckboxes).disabled = true;
+  }
+
+  // on change, if only one checkbox is checked, disable it
+  const checkboxes = document.querySelectorAll(`input[name="${checkboxName}"]`);
+  checkboxes.forEach((checkbox) => {
+    checkbox.addEventListener('change', function(event){
+      const checkedCount = document.querySelectorAll(checkedCheckboxes).length;
+
+      if (checkedCount === 1) {
+        checkedCheckbox = document.querySelector(checkedCheckboxes);
+        checkedCheckbox.disabled = true;
+      } else if (checkedCount === 2) {
+        disabledCheckbox = document.querySelector(`input[name="${checkboxName}"]:disabled`);
+        disabledCheckbox.disabled = false;
+      }
+    });
```

Comment:
```suggestion
      if (checkedCount > 1) {
        document.querySelectorAll(`input[name="${checkboxName}"]:disabled`).forEach((disabledCheckbox) => {
            disabledCheckbox.disabled = false;
        };
      } else {
        var checkedCheckbox = document.querySelector(checkedCheckboxes);
        checkedCheckbox.disabled = true;
      }
    });
```
The checkbox must all be enabled when more than one is selected no?